### PR TITLE
DOC/PLT: Add `stacked` in doc and doc example for barh and bar plot

### LINKED ISSUE #32759: keyword stacked for bar plots not documented anywhere.
#### Code Sample, a copy-pastable example if possible

```python
df.plot.bar(stacked=True)
df.plot(kind='bar', stacked=True)
```
#### Problem description

There is a keyword argument to create a stacked bar graph. This kwarg is not documented anywhere in the Pandas documentation.

#### Expected Output

The documentations for the methods `df.plot.bar` and `df.plot` should include a description of the kwd `stacked`

#### Version

Pandas 1.0.1

### PR DESCRIPTION
**Pull request by charlesdong1991**
- [x] closes #32759
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/plotting/_core.py [hunk 1]
```diff
@@ -954,6 +954,9 @@ def line(self, x=None, y=None, **kwargs):
 
     @Appender(
         """
+        stacked: bool, default is False
```
Comment (MomIsBestFriend) [inline pandas/plotting/_core.py:957]:
```suggestion
        stacked: bool, default False
```

File: pandas/plotting/_core.py [hunk 2]
```diff
@@ -1037,6 +1047,9 @@ def bar(self, x=None, y=None, **kwargs):
...
```
Comment (MomIsBestFriend) [inline pandas/plotting/_core.py:1050]:
```suggestion
        stacked: bool, default False
```

**Review by ShaharNaveh — COMMENTED** [review]
Nitpicks

File: pandas/plotting/_core.py (hunk 1 above)
Comment (charlesdong1991) [reply pandas/plotting/_core.py]: thanks!

File: pandas/plotting/_core.py (hunk 2 above)
Comment (charlesdong1991) [reply pandas/plotting/_core.py]: thanks!

File: pandas/plotting/_core.py [hunk 3]
```diff
@@ -985,6 +988,13 @@ def line(self, x=None, y=None, **kwargs):
             ...                    'lifespan': lifespan}, index=index)
             >>> ax = df.plot.bar(rot=0)
 
+        Plot stacked bar chars for the DataFrame
```
Comment (alimcmaster1) [inline pandas/plotting/_core.py:991]: Should this be ‘bar charts’?

File: pandas/plotting/_core.py [hunk 4]
```diff
@@ -1066,6 +1079,13 @@ def bar(self, x=None, y=None, **kwargs):
             ...                    'lifespan': lifespan}, index=index)
             >>> ax = df.plot.barh()
 
+        Plot stacked bar chars for the DataFrame
```
Comment (alimcmaster1) [inline pandas/plotting/_core.py:1082]: Same here?

File: pandas/plotting/_core.py (hunk 3 above)
Comment (charlesdong1991) [reply pandas/plotting/_core.py]: Oops! 😅  Nice catch!

File: pandas/plotting/_core.py (hunk 4 above)
Comment (charlesdong1991) [reply pandas/plotting/_core.py]: thanks!

**Comment by charlesdong1991** [comment]
would appreciate a lot for some reviews, thanks! ^^ @jreback @WillAyd

**Comment by WillAyd** [comment]
Can you post a screenshot of the built docs? Doesn't this place the argument out of order (i.e. after the Returns section)?

@datapythonista

**Comment by charlesdong1991** [comment]
Hi, @WillAyd thanks for your review!

You are right, it is not the right place for parameters. And I removed it because i noticed another PR #33596 has included `stacked` also. So might be better to let that PR deal with parameter doc.

So the remaining question is the code example, I think it would be nice for users to see the example. But if you think it is not needed because its quite straightforward, then let's close this PR then. I am open to hearing from your preference!

**Comment by charlesdong1991** [comment]
ping @WillAyd 

pls let me know if it is still worth it to have this one in doc? ^^

**Review by WillAyd — APPROVED** [review]
lgtm @datapythonista

**Comment by datapythonista** [comment]
Thanks @charlesdong1991

**Pull request merged**